frontend/packages/dev-console/src/components/EmptyState.tsx

```diff
@@ -52,7 +52,7 @@ const ODCEmptyState: React.FunctionComponent<Props> = ({ title, activeNamespace
             <CardFooter>
               <Link
                 className="pf-c-button pf-m-secondary"
-                to={`/deploy-image?preselected-ns=${activeNamespace}`}
+                to="/deployImage"
```

Comment:
This would break any existing links to the deploy image page. Is there a compelling reason to change the URL?

Reply:
I changed it because it moved into dev console. Not a compelling reason so I can update it.

Reply:
I realize that I made the change to match the naming style used in git import.

Reply:
I made this change but you all really need to look at it to make sure this is what you want. 

I have also left the original `deploy-image` file in place just with an `old-` prefix so that it can still be referenced while building this out.

Reply:
This change also take the PR outside of the dev console.